src/run.ts

```diff
@@ -70,6 +70,17 @@ async function run() {
         version = await getStableKubectlVersion();
     }
     let cachedPath = await downloadKubectl(version);
+
+    try {
+
+        if (!process.env['PATH'].startsWith(path.dirname(cachedPath))) {
+            core.addPath(path.dirname(cachedPath));
+        }
+    }
+    catch {

```

Comment:
What is expected to throw an error here?

Reply:
The if condition could throw an error. But since we're removing that, we can remove the try catch also.